app/indices/account_index.rb

```diff
@@ -14,6 +14,10 @@
 
   set_property field_weights: { name: 2 }
 
+  if System::Database.mysql?
+    set_property group_concat_max_len: 32_000
```

Comment:
I haven't forgotten about https://github.com/3scale/porta/pull/587#discussion_r255590681
But step by step plz, right now I am trying to fix https://github.com/3scale/porta/pull/587/files#r255618581